morpheushelper/morpheushelper.py

```diff
@@ -334,13 +340,13 @@ async def on_bot_ping(message: Message):
 register_cogs(bot, *enabled_cogs)
 
 if bot.cogs:
-    print(f"\033[1m\033[32m{len(bot.cogs)} Cog{'s' * (len(bot.cogs) > 1)} enabled:\033[0m")
+    root_logger.info("%s Cog%s enabled:", len(bot.cogs), 's' * (len(bot.cogs) > 1))
     for cog in bot.cogs.values():
         commands = ", ".join(cmd.name for cmd in cog.get_commands())
-        print(f" + {cog.__class__.__name__}" + f" ({commands})" * bool(commands))
+        root_logger.info(" + %s %s", cog.__class__.__name__, str(commands) if commands else "")
 if disabled_cogs:
-    print(f"\033[1m\033[31m{len(disabled_cogs)} Cog{'s' * (len(disabled_cogs) > 1)} disabled:\033[0m")
+    root_logger.info("%s Cog%s disabled:", len(disabled_cogs), 's' * (len(disabled_cogs) > 1))
```

Comment:
maybe it would be more readable if we use if else instead of the * operator
```python
root_logger.info("%s Cog%s disabled:", len(disabled_cogs), 's' if len(disabled_cogs) > 1 else '')
```